Packs/SymantecEndpointSecurity/Integrations/SymantecEndpointSecurity/SymantecEndpointSecurity.py

```diff
@@ -0,0 +1,459 @@
+    timestamp with the latest event time and checking if its ID is in the set of
+    suspected duplicates.
+
+    Args:
+        event_id (str): The unique identifier of the event.
+        event_time (datetime): The timestamp of the event.
+        latest_event_time (datetime): The timestamp of the last event from the last fetch.
+        events_suspected_duplicates (set): A set of event IDs suspected to be duplicates.
+
+    Returns:
+        bool: True if the event is a duplicate, False otherwise.
+    """
+    if event_time < latest_event_time:
+        return True
+    elif event_time == latest_event_time and event_id in events_suspected_duplicates:
+        return True
+    return False
+
+
+def filter_duplicate_events(
+    events: list[dict[str, str]], integration_context: dict
+) -> list[dict[str, str]]:
+    """
+    Filter out duplicate events from the given list of events.
+
+    Args:
+        events (list[dict[str, str]]): A list of event dicts, each containing 'uuid' and 'log_time' keys.
+
+    Returns:
+        list[dict[str, str]]: A list of event dicts without fear of duplication.
+    """
+    events_suspected_duplicates = set(
+        integration_context.get("events_suspected_duplicates", [])
+    )
+    latest_event_time = (
+        integration_context.get("latest_event_time", "2000-01-01T00:00:00.000Z")
+        or "2000-01-01T00:00:00.000Z"
+    )  # TODO default value
+
+    latest_event_time = datetime.strptime(
+        normalize_date_format(latest_event_time), "%Y-%m-%dT%H:%M:%SZ"
+    )
+
+    filtered_events: list[dict[str, str]] = []
+
+    for event in events:
+        if not is_duplicate(
+            event["uuid"],
+            datetime.strptime(
+                normalize_date_format(event["log_time"]), "%Y-%m-%dT%H:%M:%SZ"
+            ),
+            latest_event_time,
+            events_suspected_duplicates,
+        ):
+            event["_time"] = event["time"]
+            filtered_events.append(event)
+
+    return filtered_events
+
+
+def prepare_raw_res_and_load_json(raw_res: str) -> dict:
+    raw_res = raw_res.replace("}\n{", ",")
+    if not raw_res.startswith("["):
+        raw_res = f"[{raw_res}]"
+    return json.loads(raw_res)
+
+
+def get_events_command(client: Client, integration_context: dict):
+    events: list[dict] = []
+    next_fetch: dict[str, str] = integration_context.get("next_fetch", {})
+
+    try:
+        raw_res = client.get_events(payload=next_fetch)
+        json_res = prepare_raw_res_and_load_json(raw_res)
+    except DemistoException as e:
+        if e.res is not None and e.res.status_code == 401:
+            demisto.info(
+                "Unauthorized access token, trying to obtain a new access token"
+            )
+            raise UnauthorizedToken
+        elif e.res is not None and e.res.status_code == 410:
+            raise NextPointingNotAvailable
+        raise e
+
+    for chunk in json_res:
+        events.extend(chunk["events"])
```

Comment:
instead of creating `events` at the beginning of this method, just create it here, using `list(itertools.chain.from_iterable(chunk["events"] for chunk in json_res))`